spec.md

```diff
@@ -864,6 +867,32 @@ message GetCapacityResponse {
 }
 ```
 
+#### `ProbeController`
+
+A Controller Plugin MUST implement this RPC call.
+The Plugin SHALL assume that this RPC will be executed on the controller where the volume will be created.
+The CO SHOULD call this RPC for the controller at which it wants to create volume.
+This RPC allows the CO to probe the readiness of the Plugin on the controller where the volumes will be created.
+The Plugin SHOULD verify if it has everything it needs (binaries, kernel module, drivers, etc.) to run on that controller, and return a success if the validation succeeds.
```

Comment:
Hi @Kaixiang,

Because the Controller service is likely to be where any centralized configuration lives, I wonder if there should be some text about verifying expected configuration files/settings?